src/Geosuggest.jsx

```diff
@@ -196,7 +197,17 @@ class Geosuggest extends React.Component {
       }
     });
 
-    this.setState({suggests});
+    // check if activeSuggest is still in list
+    if (activeSuggest) {
+      const newSuggest = suggests.find(listedSuggest =>
+        activeSuggest.placeId === listedSuggest.placeId &&
+        activeSuggest.isFixture === listedSuggest.isFixture
+      );
+
+      activeSuggest = newSuggest || null;
+    }
+
+    this.setState({suggests, activeSuggest});
```

Comment:
This function is quite large already – move the check to a new function that gets called here.